contribution/testing_guide.md

```diff
@@ -25,6 +25,20 @@
 
     3. Apply security policies for testing
 
+        Beforehand, check if the KubeArmorPolicy or KubeArmorHostPolicy CRD is already applied.
+
+        ```text
+        $ kubectl explain KubeArmorPolicy
+        ```
+
+        If it's still not applied, do so.
+
+        ```text
+        $ kubectl apply -f ~/KubeArmor/pkg/KubeArmorPolicy/config/crd/bases/security.kubearmor.com_kubearmorpolicies.yaml
```

Comment:
Can you also add KubeArmorHostPolicy?
One more thing. Can you use /KubeArmor/deployments/CRD/... rather than /KubeArmor/pkg/...
those are the same things. Just for easy-to-use.

Reply:
done

Reply:
@nam-jaehyun, I was going back to it right now but thanks for doing these changes.